jlink_test.go

```diff
@@ -52,7 +52,7 @@ func testJLink(t *testing.T, context spec.G, it spec.S) {
 		var err error
 		Expect(os.Setenv("BP_JVM_JLINK_ENABLED", "true")).To(Succeed())
 
-		ctx.Layers.Path, err = ioutil.TempDir("", "jdk")
+		ctx.Layers.Path, err = os.MkdirTemp("", "jdk")
```

Comment:
Thanks for updating these too! Same comment as from the other PR. Please switch to using `t.TempDir()` https://pkg.go.dev/testing#T.TempDir. It's just a little less code and automates the cleanup.

Reply:
done!

Reply:
Thanks!